Polish/przygoda/przygoda.md

```diff
@@ -1,5 +1,5 @@
 Zew przygody wzywa cię. Czujesz, że musisz się wydostać z tego pokoju. 
-Rozglądasz się po pokoju w poszukiwaniu wyjścia. Jednak wyjścia nie ma.
+Rozglądasz się po pokoju w poszukiwaniu drzwi, jednak zauważasz tylko różowe okno w suficie. Jest za małe by przez nie wyjść.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.